Adds DRF Access Policy to permissions 3rd party section

rsinger86 opened the pull request:
*Note*: Before submitting this pull request, please review our [contributing guidelines](https://github.com/encode/django-rest-framework/blob/master/CONTRIBUTING.md#pull-requests).

## Description

Hi! This is just an addition to the 3rd party packages section of the permissions docs.

rpkilby commented:
Hi @rsinger86 - Could you also add this to the [Third Party Packages](https://github.com/encode/django-rest-framework/blob/master/docs/community/third-party-packages.md) docs as a one/two-liner? Our 3rd party package docs are a bit spread out, and a goal of mine is to consolidate them to that one page.

rsinger86 commented:
Okay! I've added it to that docs file as well. I also added another [package](https://github.com/rsinger86/drf-flex-fields) of mine, which was missing.

rpkilby commented:
Thanks!
